unearthed-arcana/2021/20210126.xml

```diff
@@ -0,0 +1,398 @@
+			<stat name="ua:hex magic:ability" value="Intelligence" inline="true"/>
+		</rules>
+	</element>
+	<element name="Wisdom (Hex Magic)" type="Racial Trait" source="Unearthed Arcana: Gothic Lineages" id="ID_WOTC_UA20210126_RACIAL_TRAIT_HEXBLOOD_HEX_MAGIC_WISDOM">
+		<compendium display="false" />
+		<description>
+			<p>Your spellcasting ability for these spells is Wisdom.</p>
+		</description>
+		<sheet display="false" />
+		<rules>
+			<stat name="ua:hex magic:ability" value="Wisdom" inline="true"/>
+		</rules>
+	</element>
+	<element name="Charisma (Hex Magic)" type="Racial Trait" source="Unearthed Arcana: Gothic Lineages" id="ID_WOTC_UA20210126_RACIAL_TRAIT_HEXBLOOD_HEX_MAGIC_CHARISMA">
+		<compendium display="false" />
+		<description>
+			<p>Your spellcasting ability for these spells is Charisma.</p>
+		</description>
+		<sheet display="false" />
+		<rules>
+			<stat name="ua:hex magic:ability" value="Charisma" inline="true"/>
+		</rules>
+	</element>
+
+	<element name="Magic Token" type="Racial Trait" source="Unearthed Arcana: Gothic Lineages" id="ID_WOTC_UA20210126_RACIAL_TRAIT_HEXBLOOD_MAGIC_TOKEN">
+		<description>
+			<p>As an action, you can harmlessly pull out one of your nails, a tooth, or a lock of hair. This token is imbued with magic until you finish a long rest. While the token is imbued in this way, you can use an action to send a telepathic message to the creature holding or carrying the token, as long as you are on the same plane of existence and are within 10 miles of it. The message can contain up to twenty-five words.</p>
+			<p class="indent">In addition, while you are within 10 miles of the token, you can use an action to enter a trance for 1 minute, during which you can see and hear from the token as if you were located where it is. While you are using your senses at the token’s location, you are blinded and deafened in regard to your own surroundings. Afterward, the token is harmlessly destroyed.</p>
+			<p class="indent">Once you create a token using this feature, you can’t do so again until you finish a long rest, at which point your missing part regrows.</p>
+		</description>
+		<sheet action="Action" usage="1/Long Rest">
+			<description>You can harmlessly pull out one of your nails, a tooth, or a lock of hair. This token is imbued with magic until you finish a long rest. While the token is imbued in this way, you can use an action to send a telepathic message to the creature holding or carrying the token, as long as you are on the same plane of existence and are within 10 miles of it. The message can contain up to twenty-five words. &#13;
+			In addition, while you are within 10 miles of the token, you can use an action to enter a trance for 1 minute, during which you can see and hear from the token as if you were located where it is. While you are using your senses at the token’s location, you are blinded and deafened in regard to your own surroundings. Afterward, the token is harmlessly destroyed.</description>
```

Comment:
I removed it since i thought it wasn't necessary for sheet description, though i'm not totally against adding it, since Racial Traits have their own box.

Reply:
I think that is one of those details that could be important to some people lol